docs/api/tutorials/Options.md

```diff
@@ -77,6 +77,38 @@ See [Video.js Responsive Option](https://videojs.com/guides/options/#responsive)
 
 The options in this section complement those of video.js, further customizing the player experience.
 
+### `srgOptions.dataProvider`
+
+***Specific to media content from SRG SSR.***
+
+Specifies a custom function returning an object representing an SRG SSR media content. This function
+must take as parameter a string representing the mediaComposition and return a JSON object
+representing the mediaComposition.
```

Comment:
I would suggest the following improvement:

```suggestion
Specifies a custom function returning an object representing an SRG SSR media content. This function
takes a single parameter: the string initially passed as the player's source , and return a JSON object
representing the mediaComposition.
```

Since the parameter doesn't have to be `a string representing the mediaComposition`, it is in the case of the example, but it could be an arbitrary value.

Reply:
Very better thank you! I just removed the extra space between `source` and `,`